### ISSUE TITLE
Bypass blocks (WCAG 2.4.1)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1235]
**Request:** Visually, websites are structured with means such as headings, columns or boxes. Thanks to this structuring, the users know what belongs together, can easily see what the website has to offer and can access the content that interests them. Users who cannot use this visual order have to rely on the structure being accessible and usable regardless of what is displayed on the screen. The use of (often invisible) area headings, jump links or HTML5 elements to mark regions is an essential requirement for this.

Page areas are accessible through headings, jump links or WAI ARIA document landmarks.  Frames and iframes have meaningful titles. 

**Issues:**
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- The page areas are sometimes not properly accessed through the labeling with` header, nav, main, aside` and `footer`. `Main` and `nav` should be used instead of the `region` and `complementary` landmarks.

2. https://slidewiki.org/search?keywords=accessibility&sort=score
- The landmarks used do not include all areas of the page correctly. All search results should be in the landmark `main`. The search and filter area should have its own landmarks (e.g. `search` and `nav`). If `nav` landmarks are used twice, they should be clearly identified by `aria-label` or `aria-labelelledby`.

3. https://slidewiki.org/user/vivien.zschammer/recommendations 
- The site has landmarks. However, the main content of the page is not identified by the `main` element.

4. https://slidewiki.org/addDeck 
- See page 3.
5. https://slidewiki.org/contactus 
- See page 3.
6. https://slidewiki.org/accessibility 
- See page 3.
7. a. https://slidewiki.org/user/vivien.zschammer/settings/profile
b. https://slidewiki.org/user/vivien.zschammer/stats 
- See page 3.
